### DISCUSSION TITLE
User Research on LXQt

### DISCUSSION THREAD

**Discussion by markbil** [lxqt/lxqt#2283] [Ideas]
I had a chance to talk to a LXQt user on DebConf22, and recorded core insights (find below). Let me know if this is of any value, and, if yes, where is a good place to store further user interviews / key user research insights.

**Persona relevant information**
- uses low-spec laptop (2 GB RAM): Lenovo 100S
- "power-user" without developer skills, i.e. likes ability to configure his interface without cli skills
- likes adaptability/customisability
- likes lightweightedness over latest UI-trends and bloat
- key-metric of concern for DE is fast booting and RAM usage (performance on low-spec machines should remain as high as possible)
- treats his computer as "cattle", not a "pet", i.e. wants LXQt to work well out of the box, i.e not needing configuration to start working. wants LXQt to come with as little weight/bloat by default (e.g. packages like libre-office are too much). Therefore he chose to only install debian + lxqt-core, and other dependencies manually (i.e. desktop manager: lightdm; window-manager: xfwm4, x-server)

**Why do you use LXQt?**
- I was looking for the most lightweight DE. my laptop is very low spec, only has 2GB of RAM (Lenovo 100S)
- I am extremely satisfied with LXQt. it's an improvement over LXDE, because it's more lightweight
- LXQt is actively  maintained, so it seems future-proof.
- Qt is okay to use by now; there used to be a non-free licencing issue, but now everything is fine. so there is no longer a reason not to use Qt.

**What would need to happen that you stop using LXQt?**
- don't include bloat. if LXQt becomes less lightweight, that would be a reason for me to leave. I require the ability to run LXQt on low-spec hardware
- developers usually have good intentions, but often with the unintented consequence of adding blaot

**Desired improvements**
- not much to say here. I am very satisfied with LXQt.
- when dragging/moving a window towards the edges, the window sometimes goes full screen even though when this is unintended by me. I know there are settings to adjust that, but I didn't want to tweak.
- When I decide to move to another computer, it would be nice ot have an easy-to-use function that allows me to export/import all my LXQt settings and configuration to a new computer
- renaming of desktop tabs is possible, but not persistent. once I reboot the custom namings are reset to the default names again (Desktop 1, Desktop 2, etc)

**Why do you not use other desktop environments?**
- XFCE is also lightweight, but user interface looks too childish. it looks like it was made for children. not really my style
- LXDE is also lightweight, but maintenance seems to be stalled; not so many changes in updates.
- GNOME3 is too dumbed-down; they decreased complexity by removing clutter and anything that is unnecessary, but the interface does not allow for configurability. they follow a scandinavian, minimalistic, Apple-like design philosophy - you do it the GNOME3 way or you don't do it at all. they offer no or only few choices in favour of reducing intellectual challenge and load; it's clearly targeted at the average, non-techy, non power-user. I e.g. I like to have my clock widget display seconds, but GNOME does not allow to configure their clock widget.
- Microsoft Windows, depends on the mouse too much. They design their interfaces so the average user can do everything by using the mouse. the keboard is only used for text input. As a power-user I like to use my keyboard for shortcuts, etc. To be fair, Microsoft has changed this philosophy recently with Visual Studio Code which addresses power-users, but in general their OS still targets average and novice users, not power-users like myself.

**Summary: core perceived values and characteristics of LXQt**
- lightweighted (can run on very low-spec hardware). key concern on RAM usage
- works well out-of-the-box, yet easily configurable/customisable
- addresses "power-users" with non-techy/developer skills (i.e. not requiring cli skills to configure/customise user interface)
- actively maintained
- minimalistic
- based on free software

**Comment by tsujan** [comment]
Interesting! They're almost my reasons for choosing LXQt too, except that, as a developer, I don't use low-spec computers.

> don't include bloat

Never.

> when dragging/moving a window

Change/configure your WM. One of the good things about LXQt is that it's WM-agnostic.

**Comment by ringo32** [comment]
system a 16gb ram pc with amd cpu/gpu   
just a linux user and supporter on another distrobution arcb-based.

I use lxqt-openbox  it just have all i need :) kwin has issues on me with azerty , i like to switch windows with keys , super+& and super+é but kwin makes éçè to a big letter so it doesnt work; and latelty xfwm4 also sort bugs.. openbox is stable in use for me :)

lxqt is pretty modular if it goes like gnome yes it would drive me away.. lxqt is as the base modular like its brother lxde so its fine :)

Personal i would see as desktop it growth a bit more in users and if things comes just step by step...not to reinvent the wheel at every release thats bad :)

genezrally i have a knack for xfce and lxqt but lxqt interest me. top3 desktops for me is lxqt / xfce and plasma.   

For me i want to use the computer and a lightweight desktop helps me to use it all.

**Comment by stefonarch** [comment]
Thanks, interesting!

>* when dragging/moving a window towards the edges, the window sometimes goes full screen even though when this is unintended by me. I know there are settings to adjust that, but I didn't want to tweak.

this is outside of LXQt, it's hard for the average user to understand what has to be configured in the WM settings.

>  * When I decide to move to another computer, it would be nice ot have an easy-to-use function that allows me to export/import all my LXQt settings and configuration to a new computer

That would be a  nice feature request! 

>* renaming of desktop tabs is possible, but not persistent. once I reboot the custom namings are reset to the default names again (Desktop 1, Desktop 2, etc) 

https://github.com/lxqt/lxqt-panel/issues/1693 probably no solution as desktop names too tided tinto XFCE sessio, there's no config file around outside of it.

**Comment by xgdgsc** [comment]
I use lxqt on linux servers that run multiple virtualbox machines. I hope there' s a server default configuration release that defaults to no screensaver/volume panel, etc ,that has no effect on server.
